index.php

```diff
@@ -41,7 +41,9 @@ public function construct() {
 	 * Adds settings options
 	 */
 	public function add_settings() {
-		require_once dirname( __FILE__ ) . '/src/Tribe/Settings_Helper.php';
```

Comment:
Alternatively, you could _namespace_ this plugin, like the extension template does: https://github.com/mt-support/tribe-ext-extension-template